app/models/user.rb

```diff
@@ -8,4 +8,6 @@ class User < ApplicationRecord
 
   has_many :subscriptions, class_name: "Hackathon::Subscription", foreign_key: "subscriber_id", inverse_of: :subscriber, dependent: :destroy
   has_many :digests, class_name: "Hackathon::Digest", foreign_key: "recipient_id", inverse_of: :recipient, dependent: :destroy
+
+  scope :admins, -> { where(admin: true) }
```

Comment:
We should already have this scope from `Privileged`.

Reply:
Sure thing! I am not familiar with `Concerns` concept but I will read about.

Reply:
Yeah, no problem! It's definitely not a common way of structuring models, but [here's the article that inspired it](https://dev.37signals.com/good-concerns/) if you wanna learn more.